### PR TITLE
kexuan5458-proposal-submission 3

### PR DESCRIPTION
**Pull request by kexuan5458**
---
A good PR should include the following parts:

- [ ] A clear title
- [ ] A meaningful message for PR, as well as its commits
- [ ] from your specific branch (***not master***) merging to teacher's `HW` branch (***not master***, again)
- [ ] excluding any irrelevant files, such as binaries, text files, or dot files
- [ ] passing tests/CI

### CODE REVIEW

**Comment by kexuan5458** [comment]
@yungyuc This is my final version of project proposal. Please have a look when you have time, thanks!

File: project/kexuan5458/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Self-Driving Car Localization
+
+## Basic Information
+
+Kalman Filter and Iterative Closest Point(ICP) algorithm are good tools in the self-driving car localization task.
```
Comment (yungyuc) [inline project/kexuan5458/README.md:5]: Which one will you focus on?  The Kalman filter or ICP, or they are coupled?

File: project/kexuan5458/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Self-Driving Car Localization
+
```
Comment (yungyuc) [inline project/kexuan5458/README.md:2]: It will be good to include the github link in the proposal.

**Review by yungyuc — CHANGES_REQUESTED** [review]
There are still conflict in the branch.  Please fix before I can merge.

File: project/kexuan5458/README.md (hunk 1 above)
Comment (kexuan5458) [reply project/kexuan5458/README.md:5]: They are coupled.

**Comment by kexuan5458** [comment]
@yungyuc Sorry, I don't know why I have this conflicts message. 
I add my project link at the end of "README.rst".
But it shows that there is a conflicting file.

**Comment by yungyuc** [comment]
> @yungyuc Sorry, I don't know why I have this conflicts message.
> I add my project link at the end of "README.rst".
> But it shows that there is a conflicting file.

Please consult the git documentation to handle the conflict.

It happens that I will be in an event nearby NYCU campus: https://sciwork.dev/sprint/2022/11early-hsinchu .  If you want a F2F discussion, feel free to stop by.

File: project/README.rst [hunk 3]
```diff
@@ -28,29 +28,23 @@ entry too.  You can request my review in the PR.
 
 .. Presentation
 .. ============
-
```
Comment (yungyuc) [inline project/README.rst:31]: Please do not make the unnecessary changes to the blank lines.  Leave a message after fixing it.

**Review by yungyuc — CHANGES_REQUESTED** [review]
